shell/browser/api/electron_api_web_contents.cc

```diff
@@ -2514,8 +2546,8 @@ void WebContents::Print(gin::Arguments* args) {
     settings.SetIntKey(printing::kSettingDpiVertical, dpi);
   }
 
-  base::ThreadPool::PostTaskAndReplyWithResult(
-      FROM_HERE, {base::MayBlock(), base::TaskPriority::USER_BLOCKING},
+  base::PostTaskAndReplyWithResult(
```

Comment:
Although upstream uses this method, its actually deprecated and will soon be replaced https://source.chromium.org/chromium/chromium/src/+/master:base/task_runner_util.h;l=34-38